Java-wasm-allocator/src/main/org/m4d/adp/allocator/AllocatorInterface.java

```diff
@@ -0,0 +1,29 @@
+package org.m4d.adp.allocator;
+
+import java.io.IOException;
+import java.lang.reflect.Field;
+import java.nio.Buffer;
+import java.nio.ByteBuffer;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.scijava.nativelib.NativeLoader;
+import org.wasmer.Instance;
+import org.wasmer.Memory;
+
+
+public class AllocatorInterface {
+
+  static {
+    Path p = Paths.get("../wasm_interface/target/release/libwasm_interface.so");
+    System.load(p.toAbsolutePath().toString());
```

Comment:
Q: If we will have multiple classes wrapping the same so (because this one is just for Allocator) is this the right place to load the so? Maybe we need to rename to WasmInterface and wrap everything in it.